packages/prosemirror-lwdita-localization/tsconfig.json

```diff
@@ -4,7 +4,11 @@
     "rootDir": "src",
     "outDir": "dist",
   },
+  "include": [
```

Comment:
Do we explicitly need this now? I am wondering how it worked before when we didn't have it?